### ISSUE TITLE
No Rares? Only Commons? REPORT RARES WITH SETTINGS HERE!

### ISSUE THREAD

**Issue by kipperjx2** [AHAAAAAAA/PokemonGo-Map#2929]
Found a rare? Report what rare along with your settings!

How many rares?

When?

Best,

**Comment by Matsu16** [comment]
Latest version of dev.

Got 2x 5 accounts on -st 16 -t 5 -sd 5 -k API -H 0.0.0.0 -P 420/421 --clear-db  

Nidoking,Vaporeon,Chansey, Dodrio, Clefairy, Warturtles. ... it works.

**Comment by kipperjx2** [comment]
You have 5 ptc accounts at once how do u do that? In the code. What is -p 420/421 mean? What is --clear-db?

**Comment by WtfJoke** [comment]
Just got a kaputo, meowth, psyduck, porgyon
runserver.py -u x... -p x -st 10 -l someloc -L de -t 10

**Comment by kipperjx2** [comment]
Looks like pple are using 5-10 accounts? Interesting? Info on how to set up maybe?

**Comment by WtfJoke** [comment]
-u user1 -u user2 -u user3.... -p passwordforall
or
-u user1 -p password1 -u user2 -p password2 ...

**Comment by Matsu16** [comment]
Read(#2546) (#2568)  Just create some accounts.  5 seems to work fine for st16.  5 accounts seems to match the -t so that works fine on -t5. 

You must have the latest version of dev with multiple account support.

**Comment by kipperjx2** [comment]
ok im tyring to figure out what version i am on i believe some version of master

**Comment by kipperjx2** [comment]
i understand u separated them on diff drives. in the environment what are the commands -p 420 and 5 on 421 stand for? the drives?

**Comment by Matsu16** [comment]
ports. 420 and 421 is what iam using,.. use the port u forwarded

**Comment by kipperjx2** [comment]
ok ive located dev build 2.1.0! great no to figue out this port thing. i do not have a port command in my line just a -h 0.0.0.0

**Comment by kipperjx2** [comment]
ive searched and found this. -P PORT, --port PORT  Set web server listening port. apparently its a capitol P. not sure how to use this haha

**Comment by Matsu16** [comment]
D:
cd Pokemon\
cd PokemonGo-Map-develop\
python runserver.py -a ptc -u account -p password -a ptc -u account -p passsword (ETC.)
-l "LOCATION" -st 16 -t 5 -sd 5 -k API HERE -H 0.0.0.0 -P 420 --clear-db

-P 420 makes 420 the port on localhost. (http://127.0.0.1:420)  so if u want to use 420 u have to forward it on your router.

**Comment by kipperjx2** [comment]
yes thank you. i understand the entire first part. so your saying that localhost is good and that your putting in the line 420 to set that port on the host. so it seams i need to learn how to forward port 420 to my router!

**Comment by Matsu16** [comment]
indeed. if u want to access ur map from  another device u can use the forwarded ip, or just install a ngrok tunnel. download ngrok. 

command ngrok http 420 and it will give u a tunnel link to access from another device.

**Comment by kipperjx2** [comment]
Thank you so much for the info, much appreciated. So let's say I just wanted to use my computer then I won't need to port forward my goal is to set 5 ptc accounts to get better scans using the setting you provided t being number of accounts -sd being the delay, st being the # of steps. Last question, what is the function of this? --clear-db

**Comment by Matsu16** [comment]
it cleans cache. just a clean map on startup. fresh. :)

**Comment by DomGrieco** [comment]
@Matsu16 I believe it clears your database, not your cache.

**Comment by kipperjx2** [comment]
i have 2 ptc accounts know and put them in like you wrote but it only said it signed into one is that normal do i need to put -a ptc every time?

**Comment by rpm12345** [comment]
just do two separate windows if you are confused

**Comment by kipperjx2** [comment]
Well I wanted them to work together in one map though, or would that still have the same effect? I am listing like this  

-a ptc -u xxxx -p xxxx  -a ptc -u xxxx -p xxxx -a ptc -u xxxx -p xxxx

Is that right or?

**Comment by clasherpekka** [comment]
Sounds like the new version is what I need but..... I have a few accounts that I want to scan a -st12 continuously. I am literally waiting 4 minutes and opening a new command prompt to begin the next scan.
Can someone quickly explain how I can create a ".bat file"? file that will open a new command prompt and enter a certain command every 4 minutes?

Each account is scanning 1 circle every 5 seconds. I think that is the most accurate

**Comment by phogetful** [comment]
Excuse my ignorance, but does the current development branch support multiple accounts in the matter that everyone is talking about?  I know you can add multiple accounts in now, but do they leapfrog one another when it scans?

Ex: If I use -t 5 and -sd 5 with 5 accounts, does account 1 take the first thread, account 2 takes the second thread? and so on until it hits the 5th account and rescans.  I'm testing it right now, but can't tell from the terminal if 1 account is doing it all or all 5.

**Comment by DomGrieco** [comment]
@phogetful Yes, that is exactly what they do.

**Comment by massaker** [comment]
Clasherpekka this is all ive been doing with a bat file for the last week and a half or so.  create a file name something.bat in the same folder as runserver.py

start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.719580, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.714081, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.708580, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.703079, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.697578, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.692078, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.686577, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.681076, -113.208703" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.711330, -113.198204" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.705830, -113.198204" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.700329, -113.198204" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.694828, -113.198204" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.711330, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.705830, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.700329, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.694828, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.689327, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.683826, -113.219200" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.703079, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.703079, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.697578, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.692078, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.686577, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.681076, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.675575, -113.229698" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.700329, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.700329, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.694828, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.689327, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.683826, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.678326, -113.240196" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.697578, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.697578, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.692078, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.686577, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.681076, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.675575, -113.250694" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.694828, -113.261192" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.689327, -113.261192" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.683826, -113.261192" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.678326, -113.261192" -st 3
start C:\Python27\python runserver.py -a ptc -u ****** -p ****** -l    "53.672825, -113.261192" -st 3

**Comment by clasherpekka** [comment]
@massaker Thanks. I will google how to create the .bat file haha!

I will try this out but does it leapfrog like **phogetful** said? 
Thanks for the reply

**Comment by massaker** [comment]
@clasherpekka 

It took awhile to find the proper gps points for each login this way it doesn't need to leapfrog around.
 
![pokemans](https://cloud.githubusercontent.com/assets/20763051/17282216/ca34de14-5760-11e6-9fc7-c7704aa46a40.png)

**Comment by rhinogator** [comment]
running 18 accounts -sd 5 -t 1 in a beehive. getting rares but scan speed considerably slower than before.

**Comment by eppcmogen** [comment]
@massaker bro how do you not get "search thread" errors with that many bots. I had something similar before but i kept getting disconnected from the 3 request per second limit with Niantic. I use 3 bots with -st 7 and its pretty good with decent coverage around my area. do u get those errors though?

**Comment by clasherpekka** [comment]
@massaker Oh I thought about that method but I did feel getting the right coordinates would be more painful. Did you find an easy way or trial an error?

@eppcmogen I assumed he had multiple accounts. Maybe?

**Comment by phogetful** [comment]
I get search errors using 5 accounts with -t 5 and -sd 5.  Testing with 4 accounts now with -t 4 and -sd 5, working flawlessly so far.

**Comment by massaker** [comment]
yeah just getting those search thread errors today.  worked perfectly before niantic cut off its arm. gonna have to play around a bit now :)

I got coordinates from guess and test to find the values then was able to write a quick c program to do the rest of math for me.

**Comment by clarkjam** [comment]
I have 5 accounts, -t 5, -sd 5, -st 5 (just around my office and immediate walking area), has been working fine for >220 loops, running on windows.
Have recently seen Dratinis, Hitmonlees, Electabuzzs, Pinsirs, Tentacruels, Bulbasaurs, Pikachus... But nothing I've got alerts set for (Snorlax, Clefairy, Dragonite, Blastoise, Venusaur, Charizard, Lapras) but that's not at all surprising really, don't think I've seen them around here ever.

**Comment by stlmikey80** [comment]
Im running 24 accounts with -t 65 and -sd 5. not seeing any rares at all..   it's finding pokemon.. but i got those hidden....

**Comment by Silvikk** [comment]
1 acc -st 9 -ar 10 

3 Lapras, 2 Scyther. Got them, when I was actually at work, but made a small break...;)

**Comment by timikx** [comment]
beehive -st 3 -sd 5 - lp 6 (91 accounts) (latest dev) and getting many rare 
http://prntscr.com/c03wmo

**Comment by sharklab** [comment]
> beehive -st 3 -sd 5 - lp 6 (91 accounts) (latest dev) and getting many rare 

How did you build the beehive?

**Comment by timikx** [comment]
with location_generator.py, but not from dev because there are missing some commas
this is fixed:
https://raw.githubusercontent.com/BaIthamel/PokemonGo-Map/95b7ec0f8689d0ee0a5e7011f178913cd267b682/Tools/Hex-Beehive-Generator/location_generator.py
(right mouse button, save)
then make accounts.csv with names  and password and run 
python location_generator.py -lat xxx -lon 1xxxx -st 5 -lp 3 --accounts c:\PokemonGo-Map-develop\accounts.csv --windows --installdir="c:\PokemonGo-Map-develop" --auth ptc -v
it make beehive.bat
also you need change config.ini and install mysql.
account you can easy make with this
https://github.com/sebastienvercammen/nintendo-ptc-account-generator/

**Comment by stlmikey80** [comment]
@timikx how should the accounts.csv look? Can you give example please.. I keep getting errors..

**Comment by WtfJoke** [comment]
@stlmikey80 just user,password
each line represents 1 account
[accounts_sample.csv.txt](https://github.com/AHAAAAAAA/PokemonGo-Map/files/394062/accounts_sample.csv.txt)

**Comment by stlmikey80** [comment]
@WtfJoke thank you

**Comment by foltincsaba** [comment]
I'm using this version which supports multiple accounts with 15 accounts:
https://github.com/shiona/PokemonGo-Map

and modified pulse_radus = 0.07 (because that version still had 100m)

running it with -st 18 -t 15 -sd 5
works perfectly. 
search completes in about 4 mins

**Comment by ramonae86** [comment]
@foltincsaba what command do you use to run the server? it looks like it doesn't like the -u -p username and password input method.

**Comment by lalib321** [comment]
when you say change config.ini you mean replace or what?? sorry a bit new lool

**Comment by timikx** [comment]
@lalib321 https://jonaharagon.github.io/PoGoMapWiki/#!mysql.md

**Comment by excitedbox** [comment]
I guess my 100 accounts 100 threads are overkill?
I am doing 5 scan delay and was doing 20 step limit in about 5:30 min. Just changed it to 10 steps and gonna test that now.

I am getting login errors mostly. 0 scan errors.

PS. I used this site to make my ptc accounts. It is not mine but it worked like a charm even though it looks scetchy as hell. http://www.dc4in.com/pokeacccreater.php

**Comment by foltincsaba** [comment]
@ramonae86 im using this command
-a ptc:user1:pass1 ptc:usrer2:pass2 google:user3:pass3 ...

**Comment by koki871** [comment]
Can anyone tell me why I am getting this error message on the "--clear-db" parameter, please?

runserver.py: error: unrecognized arguments: --clear-db

**Comment by MrStu82** [comment]
maybe the drop rate for rares has been lowered by niantic?

**Comment by ramonae86** [comment]
@foltincsaba Thanks! It works perfect!

**Comment by Danovadia** [comment]
@koki871 run it with -cd flag (instead of --clear-db)

**Comment by foltincsaba** [comment]
@ramonae86 
I read today that they increased the minimum scan time again to 10 sec. So i had to increase to 72 accounts. Creating even more accounts is kinda useless now, because login time of all of the accounts increased to about 2 mins now.

**Comment by ramonae86** [comment]
@foltincsaba I'm not quite familiar with this.....minimum scan time? Does it mean the minimum time you have to stay in one position so that the pokemon at the position pop up?
Also, do we have to change some arguments after this update? like -td or something? -sd is still 5?

**Comment by foltincsaba** [comment]
@ramonae86 yes, it's the minimum time in which the server responses to give pokemon positions.
i read it here:
https://www.reddit.com/r/pokemongo/comments/4vww6w/niantic_changed_scan_times_from_5sec_to_10sec/

we have to change to -sd 10

**Comment by ramonae86** [comment]
@foltincsaba Yes, thanks! That's what I'm doing right now. Also, do you know whether they're working on fixing the "database locked" issue when you do several instances at the same time?

**Comment by molecul** [comment]
@ramonae86 You need use [mysql](https://help.ubuntu.com/lts/serverguide/mysql.html) instead of sqlite. Configure **db-type** in **config.ini**.
